libcxx/include/experimental/__simd/simd.h

```diff
@@ -44,6 +44,11 @@ class simd {
   template <class _Up, enable_if_t<__can_broadcast_v<value_type, __remove_cvref_t<_Up>>, int> = 0>
   _LIBCPP_HIDE_FROM_ABI simd(_Up&& __v) noexcept : __s_(_Impl::__broadcast(static_cast<value_type>(__v))) {}
 
+  // generator constructor
+  template <class _Generator,
+            enable_if_t<experimental::__can_generate<value_type, _Generator>(make_index_sequence<size()>()), int> = 0>
```

Comment:
```suggestion
            enable_if_t<__can_generate<value_type, _Generator>(make_index_sequence<size()>()), int> = 0>
```
You only have to qualify function calls to avoid ADL.